app/iht/views/application/asset/household/household_overview.scala.html

```diff
@@ -55,31 +55,31 @@
                     generic_overview_table_heading_item(id = AssetsHouseholdOwnID,
                         questionText = Messages("iht.estateReport.assets.householdAndPersonalItemsOwnedByDeceased.title",deceasedName),
                         questionScreenReaderText = Messages("iht.estateReport.assets.householdAndPersonalItemsOwnedByDeceased.title",
```

Comment:
The accessibility (hidden) text needs to be updated too, e.g. on household overview it reads "Give an answer for household and personal items held in 
John Smith(M/CP)’s name" rather than "Give an answer for household and personal items owned only by John Smith(M/CP)"